### DISCUSSION TITLE
求教

### DISCUSSION THREAD

**Discussion by yuannsixb** [sun589/QQkey_Tool#20] [提问] [answered]
老哥，你的这个tools软件怎么使用啊

**Comment by sun589** [comment]
主界面是Skey，需扫码获取，二维码下面是选择对应业务，不同功能后面会写qzone(QQ空间)/qun(QQ群),qzone对应qzone.qq.com选项，qun对应qun.qq.com
key解析器就是clientkey利用器，要去盗号/木马专区获取
木马专区生成要smtp服务器,你填上邮箱密码就可以自动获取了,但大部分邮箱需要授权码,如需请手动搜索使用的邮箱+授权码获取方式

**Reply by yuannsixb** [reply]
哈哈，有点高深啊老哥。就是我只知道他qq号，密码啥的不知道，盗号的话具体怎么操作啊

**Reply by sun589** [reply]
那就用木马吧,单靠只有QQ号盗多少有点玄了-_-

**Comment by sun589** [comment, marked as answer]
[使用文档](https://github.com/sun589/QQkey_Tool/blob/main/docs/using_docs.md)

**Comment by yuannsixb** [comment]
好吧，我去试试，谢谢老哥了😂



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年11月18日 21:26 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [sun589/QQkey_Tool] 求教 (Discussion #20) |

使用文档
